vcd/config.go

```diff
@@ -66,15 +67,20 @@ type cachedConnection struct {
 	connection *VCDClient
 }
 
+type cacheStorage struct {
+	// conMap holds  Cached VDC authenticated connection
```

Comment:
```suggestion
	// conMap holds Cached VDC authenticated connection
```

Reply:
It is approved, but for reference - fixed.